Unable to build with pure backend

Trying to build with default-features=false and features=["pure"] causes error. The OpenSSL backend seems to work fine.

## Reproduction steps

1. Create a new cargo project and add `ecies={version="0.2.6", default-features=false, features=["pure"]}` to Cargo.toml
2. cargo build --release the code snippet below

## Expected behavior

The build should pass

## Actual behavior

The following error is thrown:

```
error[E0432]: unresolved import `once_cell::sync`
 --> /home/username/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ecies-0.2.6/src/config.rs:1:16
  |
1 | use once_cell::sync::Lazy;
  |                ^^^^ could not find `sync` in `once_cell`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `ecies` (lib) due to 1 previous error
```

## Code

```
use ecies::{encrypt, decrypt, utils::generate_keypair};

fn main() {
    let msg="Hello, this is a test.";
    let (sk, pk)=generate_keypair();
    let (sk, pk)=(&sk.serialize(), &pk.serialize());

    let encrypted_message=encrypt(pk, msg.as_bytes()).unwrap();

    let original=decrypt(sk, &encrypted_message).unwrap();

    let original=String::from_utf8(original).unwrap();

    println!("{original}");
    }
```

## Environment

* OS: Ubuntu Mate 22.04 64-bit
* rustc: 1.78.0
* ecies: 1.2.6

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,6 @@
 # Changelog
 
-## 0.2.1 ~ 0.2.6
+## 0.2.1 ~ 0.2.7
 
 - Support `no_std`
 - Revamp documentation

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -38,16 +38,16 @@ typenum = {version = "1.17.0", default-features = false, optional = true}
 chacha20poly1305 = {version = "0.10.1", default-features = false, optional = true}
 
 # random number generator
-getrandom = {version = "0.2.12", default-features = false}
+getrandom = {version = "=0.2.14", default-features = false}
 rand_core = {version = "0.6.4", default-features = false, features = ["getrandom"]}
 
 # configuration
-once_cell = {version = "1.19.0", default-features = false}
-parking_lot = "0.12.1"
+once_cell = {version = "1.19.0", default-features = false, features = ["critical-section"]}
+parking_lot = "0.12.2"
 
 [target.'cfg(all(target_arch = "wasm32", target_os="unknown"))'.dependencies]
 # only for js (browser or node). if it's not js, like substrate, it won't build
-getrandom = {version = "0.2.12", default-features = false, features = ["js"]}
+getrandom = {version = "=0.2.14", default-features = false, features = ["js"]}
 once_cell = {version = "1.19.0", default-features = false, features = ["std"]}
 wasm-bindgen = {version = "0.2.92", default-features = false}
 
@@ -74,12 +74,12 @@ criterion = {version = "0.4.0", default-features = false}
 hex = {version = "0.4.3", default-features = false, features = ["alloc"]}
 
 [target.'cfg(target_arch = "wasm32")'.dev-dependencies]
-wasm-bindgen-test = "0.3.37"
+wasm-bindgen-test = "0.3.42"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
-futures-util = "0.3.29"
-reqwest = "0.12.2"
-tokio = {version = "1.33.0", features = ["rt-multi-thread"]}
+futures-util = "0.3.30"
+reqwest = "0.12.4"
+tokio = {version = "1.37.0", default-features = false, features = ["rt-multi-thread"]}
 
 [[bench]]
 harness = false

